CMakeLists.txt

```diff
@@ -292,6 +292,12 @@ if(LINUX)
   set(CMAKE_SHARED_LINKER_FLAGS "-Wl,--no-as-needed")
 endif()
 
+if(WIN32 AND USE_DISTRIBUTED)
+  if(NOT DEFINED ENV{libuv_ROOT})
+    set(ENV{libuv_ROOT} $ENV{CONDA_PREFIX}\\Library)
```

Comment:
shall we have some of our Windows CI envs configured using conda install instead of `libuv_ROOT` env var?